tests/kafkatest/services/templates/kafka.properties

```diff
@@ -20,6 +20,8 @@ port=9092
 #host.name=localhost
 advertised.host.name={{ node.account.hostname }}
 #advertised.port=<port accessible by clients>
+listeners=PLAINTEXT://:9092,SSL://:9093
```

Comment:
Yes, it would be good to have cases where we only have SSL, I think. For benchmarks with multiple brokers, it would also be interesting to test the case where SSL is used for the producer/consumer, but not for inter-broker communication.